.github/workflows/build.yml

```
@@ -168,12 +168,30 @@ jobs:
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
   test:
+    name: >-
+      test (${{ matrix.scenario }}) -
+      ${{ matrix.platform }}-${{ matrix.architecture }}
     needs:
       - diagnostics
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
+        architecture:
+          - amd64
+          - arm64
+        platform:
+          - amazonlinux2023-systemd
```

You'll need to add a commit to comment out this unsupported platform:
```suggestion
          # Amazon Linux does not (yet?) provide a FreeIPA client package,
          # which is required for this Ansible role.
          # - amazonlinux2023-systemd
```